src/installer.redmod.ts

```diff
@@ -126,13 +141,36 @@ export const canonREDmodLayout = (
 };
 
 //
-// Helpers
+// Vortex
 //
 
-export const redmodInstructions = async (
+//
```

Comment:
The github diff viewer is really fucky here, so I just want to make sure about this. You basically took out the `redmodInstructions` method and included it in the `installREDmod` method, right?

Reply:
Uuh yeah, this is a special kinda diff, both unified and side by side are _useless_.

Functionally, yes, this is just combining the two — but I wanted to drop the `Optional` return. That’s actually a really good idea and would especially work in the pipeline way of doing this, but it has slightly different semantics elsewhere so I removed it until and if we come up with a more well-defined set of Instructions types

Reply:
ok, sounds good!